tuning/sft_trainer.py

```diff
@@ -174,7 +197,9 @@ def train(
             trainer_callbacks.append(cb)
 
     framework = AccelerationFrameworkConfig.from_dataclasses(
-        quantized_lora_config, fusedops_kernels_config
+        quantized_lora_config,
+        fusedops_kernels_config,
+        attention_and_distributed_packing_config,
```

Comment:
Just for my understanding, so these are all model loader augmentors that change how the model is loaded based on the acceleration framework configurations? Although padding free and multipack are both dataset augmentors? How does setting the acceleration framework here affect the dataset loading?

Reply:
you are right in saying padding free and multipack affect the dataloading, but more specifically
- padding free only requires modifications to data collation.
- multpack requires modification to dataloader


Both we handle by our [`AccelerationPatcher`](https://github.com/foundation-model-stack/fms-acceleration/blob/main/plugins/framework/src/fms_acceleration/accelerator_patcher.py), which is a component that we wrote to allow controlled replacements of the data collator and data loader.